Dockerfile.build

```diff
@@ -7,7 +7,7 @@
 # docker cp test:/dist/focalboard-server-linux-amd64.tar.gz .
 
 # build frontend
-FROM node:16.1.0 AS frontend
+FROM node:16.3.0 AS frontend
```

Comment:
We should always keep this in sync with `webapp/.nvmrc`. Can you update the same in there as well?

Reply:
done